source/base/quadrature.cc

```diff
@@ -26,11 +26,31 @@
 DEAL_II_NAMESPACE_OPEN
 
 
+namespace
+{
+  template <int dim>
+  void
+  set_tensor_basis_1d(const Quadrature<dim> &,
+                      std::vector<Quadrature<1>> &)
+  {}
```

Comment:
Hm, ok. But I think we've generally still moved away from functions that set objects by reference to functions that just return something. C++11 does that quite efficiently, and it's often easier to read.